packages/dd-trace/src/lambda/index.js

```diff
@@ -2,4 +2,11 @@
 
 const { registerLambdaHook } = require('./runtime/ritm')
 
-registerLambdaHook()
+const _DD_TRACE_DISABLED_INSTRUMENTATIONS = process.env.DD_TRACE_DISABLED_INSTRUMENTATIONS || ''
```

Comment:
with underscores since these are already defined by another file as global, but during uni testing, this file is not run there.

Reply:
Is it OK that this only applies to users who disable integrations via ENV var and not to users who disable integrations via the init config object?

Reply:
Oh, do Lambda users never touch that object and instead configure everything via ENV vars?

Reply:
If that's the case it's probably worth adding a comment.

Reply:
> Oh, do Lambda users never touch that object and instead configure everything via ENV vars?

Yeah, so this is enabled by default already, and there's no way to enable/disable this through the init config object or any other manner.

Reply:
Added a comment around it~